ocaml/xapi/cli_operations.ml

```diff
@@ -2814,6 +2814,11 @@ let vm_migrate printer rpc session_id params =
              let vdi = Client.VDI.get_by_uuid rpc session_id vdi_uuid in
              let sr = Client.SR.get_by_uuid remote_rpc remote_session sr_uuid in
              vdi,sr) (read_map_params "vdi" params) in
+         
+         let vgpu_map = List.map (fun (vgpu_uuid,gpu_group_uuid) ->
+             let vgpu = Client.VGPU.get_by_uuid rpc session_id vgpu_uuid in
+             let gpu_group = Client.GPU_group.get_by_uuid remote_rpc remote_session gpu_group_uuid in
+             vgpu,gpu_group) (read_map_params "vgpu" params) in
```

Comment:
Could be written this way (but not sure this is better) with fewer names and data flowing from top to bottom.
```
let vgpu_map = 
  read_map_params "vgpu" params
  |> List.map (fun (vgpu_uuid,gpu_group_uuid) ->
  ( Client.VGPU.get_by_uuid rpc session_id vgpu_uuid
  , Client.GPU_group.get_by_uuid remote_rpc remote_session gpu_group_uuid
  ))
```